exporter/prometheusremotewriteexporter/README.md

```diff
@@ -43,6 +43,10 @@ Example:
 exporters:
   prometheusremotewrite:
     endpoint: "http://some.url:9411/api/prom/push"
+    wal:
+        path: "DIRECTORY_HERE"
+        truncate-frequency: "Optional Frequency in seconds per time.ParseDuration https://golang.org/pkg/time/#ParseDuration, default of 1m"
```

Comment:
Explain how to choose a value for this. Alternatively, don't expose it as an option if the user does not have a good way to know how to choose it.

Reply:
It's quite the exercise to explain how to do so which is why I provide that link that allows them to learn and choose the value. It is configurable and Prometheus has the same.

Reply:
You mean this link https://golang.org/pkg/time/#ParseDuration ? That just describes the format of the field. I did not mean to describe that. I mean how does the user know what duration to use? Should I use 1 millisecond? 1 hour? 1 day? Some hints would be useful.